addons/mrp/models/mrp_production.py

```diff
@@ -495,6 +495,10 @@ def write(self, vals):
                 production.move_raw_ids.filtered(lambda m: m.state == 'draft')._action_confirm()
         return res
 
+    @api.model
+    def prepare_procurement_group(self, values):
+        return {'name': values['name']}
```

Comment:
should not be public + this kind of helper should be at the end of the file + _prepare_procurement_group_vals + self.ensure_one()

- Private attributes (_name, _description, _inherit, …)
- Default method and _default_get
- Field declarations
- Compute, inverse and search methods in the same order as field declaration
- Selection method (methods used to return computed values for selection fields)
- Constrains methods (@api.constrains) and onchange methods (@api.onchange)
- CRUD methods (ORM overrides)
- Action methods
- And finally, other business methods.